gsa/src/web/utils/testing.js

```diff
@@ -132,7 +134,9 @@ export const rendererWith = (
         <TestingGmpPropvider gmp={gmp}>
           <TestingCapabilitiesProvider capabilities={capabilities}>
             <TestingStoreProvider store={store}>
-              <TestingRouter history={history}>{ui}</TestingRouter>
+              <MockedProvider mocks={[]} addTypename={false}>
```

Comment:
Looks very good to me. I think we can extend this in future too and add something like `mocks` to the rendererWith params and pass it to the MockedProvider :-)